src/wallet/ismine.h

```diff
@@ -46,7 +46,7 @@ enum isminetype : unsigned int
     ISMINE_ENUM_ELEMENTS,
 };
 /** used for bitflags of isminetype */
-typedef uint8_t isminefilter;
+using isminefilter = std::underlying_type<isminetype>::type;
```

Comment:
This type is never stored in large quantities, so changing the size from 1 to 4 bytes is uncontroversial.